### PR TITLE
Groups

### PR DESCRIPTION
**Pull request by lub**
Hi,

I have no idea of react. But it works and I think it is correct.

Signed-off-by: Simon Körner github@lubiland.de

### CODE REVIEW

**Comment by ara4n** [comment]
oh, phew - thank you for doing this! lgtm

**Comment by ara4n** [comment]
and deployed!

File: src/components/HomePage.js [hunk 1]
```diff
@@ -212,6 +213,12 @@ export default React.createClass({
         return (msglink.match(/^[\!#]([^\/:]+?):(.+?)\/\$([^\/:]+?):(.+?)$/) && encodeURI(msglink) === msglink);
     },
 
+    isGroupValid(group) {
+        console.log(group);
```
Comment (MTRNord) [inline src/components/HomePage.js]: Are These console logs really needed? 😇

Comment (lub) [reply src/components/HomePage.js]: Not necessarily, but isMsglinkValid did the same and I thought I'd stick with it.

**Pull request merged**